compiler/cgen.nim

```diff
@@ -779,7 +779,7 @@ proc cgsym(m: BModule, name: string): Rope =
     result.addActualSuffixForHCR(m.module, sym)
 
 proc generateHeaders(m: BModule) =
-  m.s[cfsHeaders].add("\L#include \"nimbase.h\"\L")
+  m.s[cfsHeaders].add("\L#include <nimbase.h>\L") # see #13144
```

Comment:
But we don't copy this into ``/usr/include`` so it seems misplaced.

Reply:
done,  this PR also works without angle brackets